letsdrill.rb

```diff
@@ -1,13 +1,20 @@
-def get_letter_grade(integer)
- 
-  #Put your code here!
 
+def get_letter_grade(integer)
+  case integer
+  when 90..100 then 'A'
+  when 80..89  then 'B'
+  when 70..79  then 'C'
+  when 60..69  then 'D'
+  else 'F' 
+  end
 end
 
 def shortest_string(array)
-
-  #Put your code here!
-
+  return nil if array.length == 0
+  
+  array.inject do |memo, string|
```

Comment:
Since this iterator is only one line, there's a shorthand that's available to you. Can you find a way of making this iteration one line (hint: it has to do with an alternate syntax of passing in blocks)